packages/contracts/src/plugins/governance/majority-voting/token/TokenVotingSetup.sol

```diff
@@ -100,18 +101,7 @@ contract TokenVotingSetup is PluginSetup {
                 revert TokenNotERC20(token);
```

Comment:
In Line 95 we use `token != address(0)` we use this in several places.

Could we take this opporuntity to enhance readability a bit?

bool deployToken = token == address(0);

Or something, can be used in several places if not STD

Reply:
If you're not super keen on this, I prefer to leave it as it is as I am "lazy" to push this change and run the tests again lool.

Reply:
fine